var/spack/repos/builtin/packages/xtb/package.py

```diff
@@ -32,7 +34,9 @@ def meson_args(self):
         elif lapack != "openblas":
             lapack = "netlib"
 
+        lapack_opt = "-Dlapack={0}" if self.version > Version("6.6.0") else "-Dla_backend={0}"
```

Comment:
Building 6.6.0 with this recipe fails with
```
>> 9    ../spack-src/meson.build:19:0: ERROR: Unknown options: "la_backend"
```


Looking at https://github.com/grimme-lab/xtb/pull/241/files it appears to me that `la_backend` is to be _introduced_ in 6.6.1.

That would mean this logic is incorrect, and it should rather be
```
lapack_opt = "-Dlapack={0}" if self.version <= Version("6.6.0") else "-Dla_backend={0}"
```

Is that correct?

Reply:
Could you open a new issue or send a PR fixing this? Commenting in a merged PR is likely to get overlooked.

Reply:
https://github.com/spack/spack/pull/38309